packages/dev/loaders/src/glTF/2.0/Extensions/KHR_materials_pbrSpecularGlossiness.dynamic.ts

```diff
@@ -0,0 +1,13 @@
+import { GLTFLoaderExtensionRegistry } from "../glTFLoaderExtensionRegistry";

```

Comment:
That's what I was originally going to do, but because the extensions statically import the `GLTFLoader` which statically imports the `GLTFFileLoader`, it's basically not possible to have the `GLTFFileLoader`/`GLTFLoader` dynamically imported without dealing with the extensions. :/

Reply:
Talked offline, got aligned on extensions.